src/AppBundle/Entity/Report/Document.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace AppBundle\Entity\Report;
+
+use AppBundle\Entity\Report\Traits as ReportTraits;
+use JMS\Serializer\Annotation as JMS;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class Document
+{
+    const TYPE_PDF = 'pdf';
+    const TYPE_JPG = 'jpg';
+
+
+    /**
+     * @JMS\Groups({"document"})
+     *
+     * @Assert\NotBlank(message="Please choose a file", groups={"document"})
+     * @Assert\File(mimeTypes={ "application/pdf" }, groups={"document"})
+     *
+     * @var string
+     */
+    private $fileName;
+
+    /**
+     * @JMS\Groups({"document"})
+     *
+     * @var \DateTime
+     */
+    private $createdAt;
```

Comment:
You can remove this altogether and just pull in the 'use CreationAudit' trait. Which will automatically set the current date on creation